psga/sars_cov_2/nextflow.config

```diff
@@ -88,7 +85,7 @@ process {
         }
     } else if ( params.sequencing_technology == "ont" ) {
         withName:bam_to_fastq_ont {
-            container = "${env.DOCKER_IMAGE_URI_PATH}/primer-autodetection:${env.DOCKER_IMAGE_TAG}"
+            container = "${env.DOCKER_IMAGE_URI_PATH}/ncov2019-artic-nf-nanopore:${env.DOCKER_IMAGE_TAG}"
```

Comment:
bam_to_fastq processes require samtools which is no longer present in the primer-autodetection docker image.
I've reverted it to the ncov docker image, but in the future we will use a small dedicated docker image for this, reducing resource usage in the cluster (see: https://jira.congenica.net/browse/PSG-2503).